File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        nums.sort()
        l = 0
        n = len(nums)
        for r in range(1, n):
            if nums[l] == nums[r]:
                return True
            else:
                l += 1
        return False
        

        

File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int len = nums.length + nums.length;
        int[] ans = new int[len];

        int n = nums.length;

        for (int i = 0; i<len; i++){
            if (i<n){
                ans[i] = nums[i];
            }
            else{
                int z = i - n;
                ans[i] = nums[z];
            }            
        }
        return ans;
    }
}